crd/apis/gcpfirewall/v1alpha1/types.go

```diff
@@ -82,9 +82,8 @@ type GCPFirewallSpec struct {
 	// +optional
 	Ports []ProtocolPort `json:"ports,omitempty"`
 
-	// A collection of sources from which ingress traffic is allowed. If this
-	// field is missing, no ingress rules are programmed. if this field is empty,
-	// this rule allows all sources.
+	// A collection of sources and destinations from which ingress traffic is allowed.
```

Comment:
Optional nit: I feel like this could be worded better, but I don't have any great suggestions. @skmatti do you have any thoughts?

Reply:
Updated.